contents/handbook/product/releasing-as-beta.md

```diff
@@ -1,20 +1,32 @@
 ---
-title: Releasing a feature as beta
+title: Releasing early access and coming soon features
 sidebar: Handbook
 showTitle: true
 hideAnchor: true
 ---
 
-It's sometimes worth releasing big new features under a 'beta' label to set expectations with customers that this feature is still in active development, and might have some rough edges. We generally do this by making betas an opt-in experience for users via [the feature preview menu](https://app.posthog.com/settings/user-feature-previews).
+PostHog includes a variety of early access features [the feature previews section of a users' settings page](https://app.posthog.com/settings/user-feature-previews), as well as a roadmap of feature previews which are coming soon. 
 
-If you're releasing something as a beta there are some guidelines you should follow - especially if the intent is that the beta will eventually become a full product in it's own right. 
+Items in the feature previews section can be toggled on or off if users want to try a feature out. Items in the coming soon section enable users to register their interest so that we can contact them with updates. Both sections work only at the user-level and not at the org or project level. 
 
-## What can be released as a beta?
-A beta doesn't need to be perfect, but it should provide value to the user and have base elements of functionality. It doesn't need to be feature complete, but it should provide more than a mocked up front end. 
+## Releasing as coming soon
+If your team is considering building a feature in the future, you should consider adding it to the coming soon menu. This responsibility usually sits with the Team Lead, unless the project is an individual one. 
 
-Betas do not need to be performant for high-volume users and can have big bugs, but should be clearly marked as such in the UI. 
+To add an item to the coming soon, simply create a feature flag with the `concept` state in [PostHog's early access features tool](/docs/feature-flags/early-access-feature-management).
+
+Adding items to the coming soon menu early offers several advantages. It enables us to gauge interest in a new feature via sign-ups, equips our marketing teams with news they can promote to users, and ensures that betas can have sample users ready from the moment they launch. 
+
+Coming soon features can either be large or small, so use your judgement about what is of interest to users, but it should be something that you expect to work on in the next 3-6 months. 
+
+> **Moving from `Concept` to `Beta`**
+> Once you are ready to move an item from the coming soon roadmap to a beta which users can interact with, simply update the flag state from `concept` to `beta`. This will trigger an automatic notification to all subscribed users letting them know that they beta is available. All users who are subscribed to updates will be immediately opted in to the beta. 
```

Comment:
```suggestion
<CalloutBox icon="IconInfo" title="Moving from Concept to Beta" type="fyi">
  Once you are ready to move an item from the coming soon roadmap to a beta which users can interact with, simply update the flag state from `concept` to `beta`. This will trigger an automatic notification to all subscribed users letting them know that they beta is available. All users who are subscribed to updates will be immediately opted in to the beta. 
</CalloutBox>
```